pkg/chains/formats/intotoite6/intotoite6.go

```diff
@@ -151,6 +152,22 @@ func GetSubjectDigests(tr *v1beta1.TaskRun, logger *zap.SugaredLogger) []intoto.
 		}
 	}
 
+	mvns := artifacts.ExtractIntotoSignableTargetFromResults(tr, logger)
+	for i, obj := range mvns {
+		if m, ok := obj.(*artifacts.StructuredSignable); ok {
+			splits := strings.Split(m.Digest, ":")
+			logger.Warnf("Split %s %s", splits[0], splits[1])
```

Comment:
remove?

Reply:
Which part are you suggesting for removal?

Reply:
the log - this was taken care of!